init-tools.cmd

```diff
@@ -61,10 +59,29 @@ if NOT exist "%DOTNET_LOCAL_PATH%" (
 
 :afterdotnetrestore
 
+:: find global package cache if no cache path supplied
```

Comment:
Why are the windows and non-windows scripts different here?  Why don't they have the same behavior?

Reply:
> The build.sh update isn't done yet as I wanted to gather early feedback if that path is the right one to take.